### PR TITLE
[cluster-autoscaler] chore: update flag `max-node-provision-time`

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-71471

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
max provision time set to 1h by default 
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: stable/cluster-autoscaler/values.yaml [hunk 1]
```diff
@@ -88,6 +88,12 @@ extraArgs:
   # scale-down-unneeded-time: 10m
   # skip-nodes-with-local-storage: false
   # skip-nodes-with-system-pods: true
+  # How long to wait before we give up provisioning a new node. We shouldn't ever give up
+  # and this is controversial as we should warn a user that we have given up, for now set it up to an hour at
+  # least and we should have metrics that alert the user when this thresh-hold of an hour is hit.
```
Comment (samvantran) [inline stable/cluster-autoscaler/values.yaml:93]:
```suggestion
  # The maximum amount of time cluster-autoscaler will attempt to provision a new node.
```

**Review by samvantran — COMMENTED** [review]
This is a public repo, let's stick to comments that help the user understand the config and have the conversation around what we _should_ do internal to the company.

File: stable/cluster-autoscaler/values.yaml (hunk 1 above)
Comment (alejandroEsc) [reply stable/cluster-autoscaler/values.yaml:93]: sure thing.

**Review by samvantran — APPROVED** [review]

File: stable/cluster-autoscaler/values.yaml [hunk 2]
```diff
@@ -88,6 +88,10 @@ extraArgs:
   # scale-down-unneeded-time: 10m
   # skip-nodes-with-local-storage: false
   # skip-nodes-with-system-pods: true
+  # The maximum amount of time cluster-autoscaler will attempt to provision a new node.
+  max-node-provision-time: 60m0s
+  # Maximum time from last recorded successful autoscaler run before automatic restart
```
Comment (shaneutt) [inline stable/cluster-autoscaler/values.yaml:93]: Thanks for documenting this one too while you're at it.

**Review by shaneutt — APPROVED** [review]

**Review by armandgrillet — APPROVED** [review]

**Review by hectorj2f — CHANGES_REQUESTED** [review]
Please, let's justify this change before applying it. I understood we agreed on spotting this scenario in the logs first.

**Comment by alejandroEsc** [comment]
> Please, let's justify this change before applying it. I understood we agreed on spotting this scenario in the logs first.

This is why I placed it in do not merge. I wanted to also conduct some testing with this change.

**Comment by hectorj2f** [comment]
@alejandroEsc I think it was a mismatch in terms of timing about these comments, 👍🏻 .

**Comment by alejandroEsc** [comment]
closing for now.

**Pull request closed without merging**